controllers/utils/multicloud_util.go

```diff
@@ -136,14 +140,18 @@ var EnsureInstallPlanIsApproved = func(
 		if installPlan.Spec.Approved {
 			multiCloudLog.Info("InstallPlan has already been approved",
 				"InstallPlan", installPlan.ObjectMeta.Name, "namespace", installPlan.ObjectMeta.Namespace)
-			return InstallPlanCannotBeApproved, nil
+			return InstallPlanWasApproved, nil
```

Comment:
This is still not accurate as TALO didn't create a managedClusterAction to approve the InstallPlan.
I think the best approach here would be to add another return value, InstallPlanAlreadyApproved and return that instead.